speechbrain/utils/checkpoints.py

```diff
@@ -485,7 +486,12 @@ def __init__(
         self.allow_partial_load = allow_partial_load
 
     def add_recoverable(
-        self, name, obj, custom_load_hook=None, custom_save_hook=None,
+        self,
+        name,
+        obj,
+        custom_load_hook=None,
+        custom_save_hook=None,
+        is_optional=False,
```

Comment:
I wonder if `optional_load` might be a better name, to make it clear exactly what may or may not happen.

Reply:
Agreed. I changed to `optional_load`.